manifests/cron.pp

```diff
@@ -10,11 +10,21 @@
   $logrotate_path = $::logrotate::logrotate_bin
 
   if $name == 'hourly' {
-    $logrotate_arg = "${::logrotate::rules_configdir}/hourly"
+    $logrotate_conf = "${::logrotate::rules_configdir}/hourly"
   } else {
-    $logrotate_arg = $::logrotate::logrotate_conf
+    $logrotate_conf = $::logrotate::logrotate_conf
```

Comment:
please don't use topscope variables. Instead use $logroate::... (omit the :: after the $)